### PR TITLE
add splash screen

### LINKED ISSUE #32: Splash screen
With logo and the ability to pass some text.

For:
- Loading indicator
- For about page with version and build number

### PR DESCRIPTION
**Pull request by DasProffi**
closes #32

Testing Information:
- Test with android <11
- Test with android >=11

### CODE REVIEW

**Comment by Florian-Sabonchi** [comment]
```E/AndroidRuntime(31533): FATAL EXCEPTION: main
E/AndroidRuntime(31533): Process: de.helpwave.helpwave, PID: 31533
E/AndroidRuntime(31533): java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{de.helpwave.helpwave/de.helpwave.helpwave.MainActivity}: java.lang.ClassNotFoundException: Didn't find class "de.helpwave.helpwave.MainActivity" on path: DexPathList[[zip file "/data/app/~~w5DIFd99Wf0Lz1d0wc51Yw==/de.helpwave.helpwave-c55BETZAbf6KrHeCjisgIw==/base.apk"],nativeLibraryDirectories=[/data/app/~~w5DIFd99Wf0Lz1d0wc51Yw==/de.helpwave.helpwave-c55BETZAbf6KrHeCjisgIw==/lib/arm64, /data/app/~~w5DIFd99Wf0Lz1d0wc51Yw==/de.helpwave.helpwave-c55BETZAbf6KrHeCjisgIw==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
E/AndroidRuntime(31533): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3583)
E/AndroidRuntime(31533): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3814)
E/AndroidRuntime(31533): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
E/AndroidRuntime(31533): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
E/AndroidRuntime(31533): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
E/AndroidRuntime(31533): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2309)
E/AndroidRuntime(31533): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/AndroidRuntime(31533): 	at android.os.Looper.loopOnce(Looper.java:201)
E/AndroidRuntime(31533): 	at android.os.Looper.loop(Looper.java:288)
E/AndroidRuntime(31533): 	at android.app.ActivityThread.main(ActivityThread.java:7904)
E/AndroidRuntime(31533): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(31533): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
E/AndroidRuntime(31533): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)
E/AndroidRuntime(31533): Caused by: java.lang.ClassNotFoundException: Didn't find class "de.helpwave.helpwave.MainActivity" on path: DexPathList[[zip file "/data/app/~~w5DIFd99Wf0Lz1d0wc51Yw==/de.helpwave.helpwave-c55BETZAbf6KrHeCjisgIw==/base.apk"],nativeLibraryDirectories=[/data/app/~~w5DIFd99Wf0Lz1d0wc51Yw==/de.helpwave.helpwave-c55BETZAbf6KrHeCjisgIw==/lib/arm64, /data/app/~~w5DIFd99Wf0Lz1d0wc51Yw==/de.helpwave.helpwave-c55BETZAbf6KrHeCjisgIw==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
E/AndroidRuntime(31533): 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
E/AndroidRuntime(31533): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
E/AndroidRuntime(31533): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
E/AndroidRuntime(31533): 	at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:95)
E/AndroidRuntime(31533): 	at androidx.core.app.CoreComponentFactory.instantiateActivity(CoreComponentFactory.java:45)
E/AndroidRuntime(31533): 	at android.app.Instrumentation.newActivity(Instrumentation.java:1333)
E/AndroidRuntime(31533): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3570)
E/AndroidRuntime(31533): 	... 12 more```

**Comment by DasProffi** [comment]
How did you test it? Worked for me on android 11 and 13

**Comment by Florian-Sabonchi** [comment]
> How did you test it? Worked for me on android 11 and 13

Pulled your Branch and compiled in android Studio

**Comment by Florian-Sabonchi** [comment]
Coud be a Problem with Graphen OS i dont know bevore approving we need to verify that and also test on IOS

**Comment by DasProffi** [comment]
the mobile brnach is android and ios only, web crashes, becasue of flutter_osm

File: pubspec.yaml
Comment (use-to) [inline pubspec.yaml:86]: TYPOOO :D

**Review by use-to — CHANGES_REQUESTED** [review]

**Review by use-to — APPROVED** [review]

**Comment by use-to** [comment]
Waiting for @MaxSchaefer `s approval!

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**
